logging/log.go

```diff
@@ -359,6 +359,7 @@ func Base() Logger {
 // NewLogger returns a new Logger logging to out.
 func NewLogger() Logger {
 	l := logrus.New()
+	l.ExitFunc = func(code int) {}
```

Comment:
i might remove this, I don't think it was meant to be left in.

Reply:
I have removed this, I did this as a convenience when I wrote some unit tests and forgot to take it out. Unit tests which call fatal can now use TestingLogWithoutFatalExit to register exit handlers without fear of `os.Exit`